samples/DevTestLabs/GatewaySample/arm/gateway/Deploy-AzureResourceGroup.ps1

```diff
@@ -0,0 +1,260 @@
+# .\Deploy-AzureResourceGroup.ps1 -ResourceGroupLocation EastUS -Reset
```

Comment:
Formatting. Make sure to make all functions use same formatting for parameters, etc, specially since these are samples.

Reply:
This the the basic Deploy-AzureResourceGroup.ps1 that comes with Azure ResourceGroup projects in Visual Studio with a few lines added to the end.  Would you like to see the detailed summary information at the top as I did with the other Azure PowerShell scripts?

Reply:
No, I suppose it would be fine. Perhaps, in future, we could add origin information.

Reply:
Sure, I can do that.  Thanks.